ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -20,41 +21,48 @@ namespace ntbs_service.DataMigration
 {
     public interface INotificationMapper
     {
-        Task<IEnumerable<IList<Notification>>> GetNotificationsGroupedByPatient(List<string> notificationId);
-        Task<IEnumerable<IList<Notification>>> GetNotificationsGroupedByPatient(DateTime rangeStartDate, DateTime endStartDate);
+        Task<IEnumerable<IList<Notification>>> GetNotificationsGroupedByPatient(PerformContext context,
```

Comment:
Stylistically I prefer if arguments are going to go to new lines for them all to be on new lines - but couldn't be happier to be ignored on this one.

Reply:
I actually agree, this lazily left it to Rider!
gotta love the phrase "couldn't be happier to be ignored" though! 😂